content/articles/how-to-add-support-for-spatial-data-in-your-database-using-geoalchemy-and-spatialite/index.md

```diff
@@ -0,0 +1,397 @@
+---
+layout: engineering-education
+url: /how-to-add-support-for-spatial-data-in-your-database-using-geoalchemy-and-spatialite/
+title: How To Add Support For Spatial Data in your Database Using GeoAlchemy and Spatialite
+description: This tutorial will show a step-by-step guide on how to add support to store and query spatial data for their database in their development environment.
+author: paul-asalu
+topics: []
+excerpt_separator: <!--more-->
+images:
+  - url: /engineering-education/how-to-build-a-clock-with-javascript-and-svg/hero.png
+    alt: Building a clock using JavaScript and SVG
+---
+
+When building certain kinds of applications, it is important to know what kind of data you'll be working with, so you can make adequate preparations to hold and process them. One such instance is when you need to work with "spatial data" - a typical example of which might be: storing location ( in terms of longitude and latitude values).
+<!--more-->
+In this article, we will use Sqlite and its spatial extension to create a database that supports spatial data read and write operations, for a simple flask server.
+
+
+### Table of Contents
+
+- [Introduction](#introduction)
+- [Building A Simple Web Server](#building-a-simple-web-server)
+- [Adding Support For Spatial Data](#adding-support-for-spatial-data)
+- [Conclusion](#conclusion)
+
+
+### Introduction
+
+When building certain kinds of applications, it is important to know what kind of data you'll be working with, so you can make adequate preparations to hold and process them. One such instance is when you need to work with "spatial data" - a typical example of which might be: storing location ( in terms of longitude and latitude values). In addition, you may want to be able to make queries based on these kinds of information, for example, you could imagine querying your database for "10 closest entities to a given location (lon,lat)".
```

Comment:
`When building certain kinds of applications, it is important to know what kind of data you'll be working with, so you can make adequate preparations to hold and process them. One such instance is when you need to work with "spatial data" - a typical example of which might be: storing location ( in terms of longitude and latitude values).`

This is redundant